.github/workflows/update_galata_references.yaml

```
@@ -98,7 +98,7 @@ jobs:
         working-directory: ui-tests
         env:
           PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1
-        run: jlpm install
+        run: yarn playwright install
```

We still need to install npm dependencies: 

```suggestion
        run: |
          jlpm install
          yarn playwright install
```

---

Oops! Sorry 😅